npm/pkg/transport/pb/transport.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package pb;
+option go_package = "github.com/Azure/azure-container-networking/npm/pkg/transport/pb;pb";
+
+import "google/protobuf/struct.proto";
+
+// DataplaneEvents represents the Service RPC exposed by the gRPC server.
+service DataplaneEvents{
+	rpc Connect(DatapathPodMetadata) returns (stream Events);
+}
+
+// DatapathPodMetadata is the metadata for a datapath pod
+message DatapathPodMetadata {
+  string pod = 1; // Daemonset Pod ID
+  string node = 2; // Node name
+  enum APIVersion {
```

Comment:
@vakalapa making this an enum so that we default to version 1 if not specified. Easier to track versions with enum. WDYT?

Reply:
that makes sense!